.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Gem
```

Comment:
I wanted to have a discussion about we manage versioning within our private forks. Not sure what the best practice is.

We need to begin publishing our gems to Github gem server so that we can access them more elegantly in CI workflows.